app/models/events.js

```diff
@@ -27,8 +27,6 @@ function createEvent(params, callback) {
         '   ($1, $2, $3, tstzrange($4, $5))'
     ].join('\n');
 
-    console.log(query);
```

Comment:
@node-calendar/backend I think we need custom centralized logging system. This system may output all logs to files and standard output also. This logging system will have different message levels (e. g. `warning`, `error`, `debug`)